tests/storages_tests/journal_tests/test_storage.py

```diff
@@ -0,0 +1,102 @@
+import os
```

Comment:
[nit] Might be a little bit better to rename this into `test_log_compatibility.py`.